web/src/TabNav.tsx

```diff
@@ -49,8 +51,22 @@ class TabNav extends PureComponent<NavProps> {
             </Link>
           </li>
         </ul>
-      </nav>
-    )
+      </span>,
+    ]
+
+    if (this.props.sailUrl) {
+      spans.push(
+        <span className="TabNav-spacer" key="spacer">
+          &nbsp;
+        </span>
+      )
+      spans.push(
+        <span className="sail-url" key="sail-url">
+          Share this view! <a href={this.props.sailUrl}>{this.props.sailUrl}</a>
```

Comment:
If it's not vital to see the link itself, perhaps "Share this view" can just have the link...thus taking up less horizontal space.

Reply:
totes! don't wanna wait for tests again but i'll change in my next pass